admin documentation (#198)
maykinmedia/objects-api: docs/admin/index.rst

.. _admin_index:

Admin interface
===============

> Both the Objects API and the Objecttypes API have the admin interface which

Reviewer: ```suggestion
Both the Objects API and the Objecttypes API have an admin interface that
```

> allows to configure APIs and to manage objects and related object types.

Reviewer: ```suggestion
allows you to configure the APIs and to manage objects and related object types.
```

> Below you can find tutorials how to use admin interface to manage APIs.

Reviewer: ```suggestion
Below, you can find tutorials how to use the admin interface to manage the APIs.
```
